.github/dependabot.yml

```diff
@@ -0,0 +1,25 @@
+version: 2
+updates:
+  # Enable version updates for GitHub Actions
+  - package-ecosystem: 'github-actions'
+    # or GitHub Actions, set the directory to / to check for workflow files in .github/workflows.
+    directory: '/'
+    schedule:
+      interval: 'daily'
+
+  # Enable version updates for python
+  - package-ecosystem: 'pip'
+  # Look for `requirements.txt` file
+    directory: '/'
+    target-branch: "master"
+    schedule:
+      interval: 'daily'
```

Comment:
I'm not sure we want dependabot making changes directly on `master`. At least, we aren't doing this for any of our Azure Functions repos. We only automate the depenabot updates to `develop` and then merge `develop` to `master` on a manual Release process.

Reply:
That's a good point. I thought it might make sense to keep both develop and master up to date, since unlike the Azure repos, we aren't actually "deploying" anything.

For the SSOT repositories split into versions, would we then create a develop branch for every version?